config.json

```
@@ -694,6 +694,14 @@
         "practices": [],
         "prerequisites": [],
         "difficulty": 4
+      },
+      {
+        "slug": "parallel-letter-frequency",
+        "name": "Parallel Letter Frequency",
+        "uuid": "61b5fc26-dccd-4ed2-a15e-723273632d5d",
+        "practices": [],
+        "prerequisites": [],
+        "difficulty": 4
```

```suggestion
        "difficulty": 5
```

I might bump this up to a 5, what do you think?

---

I think going up to 5 is reasonable, especially because of the Unicode test case.